### ISSUE TITLE
filter runs by scalar value

### ISSUE THREAD

**Issue by dbieber** [tensorflow/tensorboard#1569]
Feature request:
Support filtering runs by scalar value.

For example, I have a scalar summary called accuracy that I care about. I run a sweep of 300 runs, and I'm browsing the results in tensorboard. I'd like to be able to filter the runs to show only those with accuracy >= 0.80. Currently I can filter by directory name, which is helpful, but it can still be really difficult to filter out just the low accuracy runs.

**Comment by nfelt** [comment]
I think this use case could be addressed by work in progress by @erzel on a hyperparameters dashboard which would provide dedicated support for viewing runs in a "sweep" evaluation context where you can filter by arbitrary scalar-summary metrics (like accuracy).

**Comment by GalOshri** [comment]
The dashboard mentioned above has been enabled in TensorBoard and there is a tutorial about this [here](https://github.com/tensorflow/tensorboard/blob/master/docs/r2/hyperparameter_tuning_with_hparams.ipynb).

Does this help? Alternatively, we might want to enable similar filtering in the scalars dashboard as well.

**Comment by dbieber** [comment]
I'm getting around to trying this now. Is the tutorial you linked still the best starting point?

**Comment by wchargin** [comment]
Yep, that’s the one.

**Comment by yulonglin** [comment]
> The dashboard mentioned above has been enabled in TensorBoard and there is a tutorial about this [here](https://github.com/tensorflow/tensorboard/blob/master/docs/r2/hyperparameter_tuning_with_hparams.ipynb).
> 
> Does this help? Alternatively, we might want to enable similar filtering in the scalars dashboard as well.

Has the tutorial been moved to another link? I am trying to filter runs based on a scalar value too

**Comment by bmd3k** [comment]
The tutorial was moved [here](https://github.com/tensorflow/tensorboard/blob/master/docs/hyperparameter_tuning_with_hparams.ipynb).
